src/stream/src/lib.rs

```diff
@@ -55,14 +59,37 @@ pub mod task;
 #[cfg(test)]
 risingwave_expr_impl::enable!();
 
+tokio::task_local! {
+    pub static CONFIG: Arc<StreamingConfig>;
+}
```

Comment:
As `StreamingConfig` is a global config, why using a `task_local` variable instead of a static variable e.g. `LazyLock`?

Reply:
I was suddenly confused at that time🥵. Now changed to a global variable initialized when compute node starting.